Sources/Widgets/Representations/RectangleContextRepresentation/index.js

```diff
@@ -55,6 +44,7 @@ function vtkRectangleContextRepresentation(publicAPI, model) {
     }
 
     const list = publicAPI.getRepresentationStates(inData[0]);
+    // FIXME: support list <> 1.
```

Comment:
We should check for at least one available state, otherwise indicate some error.